src/main/java/org/craftercms/studio/impl/v1/repository/alfresco/AlfrescoContentRepository.java

```diff
@@ -1069,6 +1099,11 @@ protected Session getCMISSession(boolean alfrescoCMIS) {
         Repository repository = repositories.get(0);
         parameter.put(SessionParameter.REPOSITORY_ID, repository.getId());
         Session session = sessionFactory.createSession(parameter);
+        //session.getDefaultContext().setCacheEnabled(false);
```

Comment:
Please dont comment out code. just delete it.  if it needs to be here for some reason there needs to be a comment why